servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -70,69 +70,31 @@ void RenderForwardMobile::ForwardIDStorageMobile::map_forward_id(RendererRD::For
 	forward_id_allocators[p_type].last_pass[p_id] = p_last_pass;
 }
 
-void RenderForwardMobile::fill_push_constant_instance_indices(SceneState::InstanceData *p_instance_data, const GeometryInstanceForwardMobile *p_instance) {
-	uint64_t current_frame = RSG::rasterizer->get_frame_number();
+void RenderForwardMobile::fill_push_constant_instance_indices(SceneState::InstanceData* p_instance_data, const GeometryInstanceForwardMobile* p_instance) {
+    uint64_t current_frame = RSG::rasterizer->get_frame_number();
```

Comment:
Also, you don't use the code style from [.clang-format](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#c-and-objective-c), it's mandatory.
```suggestion
void RenderForwardMobile::fill_push_constant_instance_indices(SceneState::InstanceData *p_instance_data, const GeometryInstanceForwardMobile *p_instance) {
	uint64_t current_frame = RSG::rasterizer->get_frame_number();
```